plugins/poll/lib/polls_validator.rb

```diff
@@ -80,10 +106,10 @@ def valid_number_of_options?(poll)
     def valid_multiple_choice_settings?(poll)
       if poll["type"] == "multiple"
         options = poll["options"].size
-        min = (poll["min"].presence || 1).to_i
+        min = (poll["min"].presence || 0).to_i
```

Comment:
Since this is a multiple choice poll, we only allow at least 1 choice. So that was correct before.